contrib/misc/bareos_pam_integration/README.rst

```diff
@@ -126,7 +133,29 @@ You may want to add following section to your :file:`/etc/bareos-webui/directors
    pam_console_name     = "pam-webui"
    pam_console_password = "secret"
 
-Now you should be able to login to the WebUI using PAM users.
+PAM users require a dedicated User Resource, see https://docs.bareos.org/master/Configuration/Director.html#user-resource .
+
+A working example User Resource configuration for a WebUI user named Alice
+looks like following.
+
+:file:`/etc/bareos/bareos-dir.d/user/Alice.conf`
+
+::
+
+   User {
+      Name = "Alice"
```

Comment:
I would prefer `alice` instead of `Alice`. If think must of the Bareos documentation uses lowercase names.

Reply:
Changed.